azure-pipelines-dependency-graph-testing.yml

```diff
@@ -180,7 +180,8 @@ stages:
       - job: compatibility_matrix
         condition: eq(stageDependencies.scope_check.changes.outputs['CheckChanges.GEChanged'], true)
         variables:
-          GE_pytest_opts: '--no-sqlalchemy'
+          - group: 'common_secrets'
+          # GE_pytest_opts: '--no-sqlalchemy'  # TODO: resolve using this with `- group`
```

Comment:
I commented this `GE_pytest_opts` key in order to get the `coveralls` step working (which it now does 🎊 ), but I assume we need to add this back.

One solution could be to set the `common_secrets` available at the stage level, but that's probably sub-optimal.